openfe/protocols/openmm_utils/multistate_analysis.py

```diff
@@ -363,10 +364,10 @@ def get_forward_and_reverse_analysis(
 
         forward_reverse = {
             'fractions': np.array(fractions),
-            'forward_DGs': unit.Quantity.from_list(forward_DGs),
-            'forward_dDGs': unit.Quantity.from_list(forward_dDGs),
-            'reverse_DGs': unit.Quantity.from_list(reverse_DGs),
-            'reverse_dDGs': unit.Quantity.from_list(reverse_dDGs)
+            'forward_DGs': Quantity.from_list(forward_DGs),  # type: ignore
```

Comment:
These are all because the pyi from openff-units doesn't define the method. Manually adding it fixes the problem. Raising the issue here: https://github.com/openforcefield/openff-units/issues/127